rasa/core/utils.py

```diff
@@ -36,7 +36,7 @@ def configure_file_logging(logger_obj: logging.Logger, log_file: Optional[Text])
         return
 
     formatter = logging.Formatter("%(asctime)s [%(levelname)-5.5s]  %(message)s")
-    file_handler = logging.FileHandler(log_file)
+    file_handler = logging.FileHandler(log_file, encoding='utf-8')
```

Comment:
```suggestion
    file_handler = logging.FileHandler(log_file, encoding="utf-8")
```
Code formatting is failing. You should use double quotes.